cinema/management/wait_for_db.py

```diff
@@ -0,0 +1,20 @@
+import time
+
+from django.core.management import BaseCommand
+from django.db import connections, OperationalError
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database...")
+        connection = None
+        while not connection:
+            try:
+                connection = connections["default"]
+                connection.cursor().execute("SELECT 1")
```

Comment:
Instead of hardcoding the `default` database alias, it would be better to use `django.db.DEFAULT_DB_ALIAS`. This will make your code more flexible and maintainable.